psalm-baseline.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="4.1.1@16bfbd9224698bd738c665f33039fade2a1a3977">
+  <file src="src/Mutator/Regex/PregMatchRemoveCaret.php">
+    <ImpureFunctionCall occurrences="1">
```

Comment:
Unfortunately, `thecodingmachine/safe` package does not support Psalm's annotations yet, and functions are missing `@pure` annotation, that's why we have issues reported by Psalm.

Tried https://github.com/hectorj/safe-php-psalm-plugin, but it doesn't work properly, so for now it's ok to suppress them